extensions-jvm/flink/deployment/pom.xml

```diff
@@ -34,6 +34,18 @@
             <groupId>org.apache.camel.quarkus</groupId>
             <artifactId>camel-quarkus-core-deployment</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-jackson-deployment</artifactId>
+        </dependency>
+         <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-netty-deployment</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.dataformat</groupId>
+            <artifactId>jackson-dataformat-csv</artifactId>
+        </dependency>
```

Comment:
Why `jackson-dataformat-csv` is added here?

Reply:
Ahh you are right its not required for JVM mode let me fix that.

Reply:
@zhfeng fixed, can you take look now? thanks!